src/server/certificates/controller.test.js

```diff
@@ -94,6 +95,125 @@ describe('#certificatesController', () => {
   })
 })
 
+describe('API calls for GET/PUT/DELETE', () => {
+  /** @type {Server} */
+  let server
+  let mockGetCertificate, mockPutCertificate, mockDeleteCertificate
+  const apiHeaderKey = config.get('apiAuth')
+  const result = {
+    certNumber: 'GBR-2024-CC-123A4BC56',
+    timestamp: '12 MAY 2024',
+    status: 'COMPLETE',
+    isValid: true
+  }
+
+  beforeAll(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  beforeEach(() => {
+    mockGetCertificate = jest.spyOn(Certificates, 'getList')
+    mockPutCertificate = jest.spyOn(Certificates, 'uploadCertificateDetails')
+    mockDeleteCertificate = jest.spyOn(Certificates, 'removeDocument')
+    mockGetCertificate.mockResolvedValue(result)
+    // mockPutCertificate.mockResolvedValue(result)
+    // mockDeleteCertificate.mockResolvedValue(result)
+  })
+
+  afterEach(async () => {
+    await server.stop()
+    mockGetCertificate.mockRestore()
+    mockPutCertificate.mockRestore()
+  })
+
+  test('Should call the api certficates api successfully', async () => {
+    const { statusCode } = await server.inject({
+      method: 'GET',
+      headers: {
+        'x-api-key': apiHeaderKey
+      },
+      url: '/api/certificates'
+    })
+
+    expect(statusCode).toBe(200)
```

Comment:
please add a check to see that a list of items are actually returned

Reply:
Thanks, @alexwbbr - Updated now.